tests/test_symbolics.py

```diff
@@ -354,25 +355,25 @@ def test_multibounds_op(self):
         assert fnorm == fnorm2
 
     @pytest.mark.parametrize('op, expr, assumptions, expected', [
-        ([min, '[a, b, c, d]', '[]', 'MIN(a, MIN(b, MIN(c, d)))']),
-        ([max, '[a, b, c, d]', '[]', 'MAX(a, MAX(b, MAX(c, d)))']),
+        ([min, '[a, b, c, d]', '[]', 'Min(a, Min(b, Min(c, d)))']),
```

Comment:
Being C macros, it would be nicer if they were all uppercase letters 😬 

see here https://stackoverflow.com/questions/20269669/define-why-uppercase for example